Changelog.md

```diff
@@ -11,7 +11,19 @@ Breaking changes:
  * Standard JSON Interface: Add option to disable or choose hash method between IPFS and Swarm for the bytecode metadata.
  * Syntax: ``push(element)`` for dynamic storage arrays do not return the new length anymore.
  * Type checker: Resulting type of exponentiation is equal to the type of the base. Also allow signed types for the base.
+ * Natspec JSON Interface: Support multiple ``@return`` statements in dev documentation to behave like named parameters.
 
+Language Features:
+
+
+Compiler Features:
+
+
+Bugfixes:
+
+
+
+### 0.5.13 (unreleased)
```

Comment:
There's an additional headline that slipped down there: https://github.com/ethereum/solidity/pull/7534/files#diff-9b8fab691c00b9e5380b19ce882f3271R44.